### PR TITLE
[PWA-2006] Contact Us Block

### PR DESCRIPTION
**Pull request by mikhaelbois**
## Description

This is a follow-up of https://github.com/magento/pwa-studio/pull/3538 where the Contact Us Page and form are created.
This tasks uses two CMS Blocks to complete the page content.
`contact-us-banner` is used for the page's banner.
`contact-us-sidebar` is used for the page's sidebar.
Those CMS Blocks are created in: https://github.com/magento-commerce/venia-sample-data-modules/pull/13

## Related Issue

Closes https://jira.corp.magento.com/browse/PWA-2006.

#### Test scenario(s) for direct fix/feature

You will have to test locally with the Venia Sample Data package installed in your backend.
Checkout the PR branch for the backend from https://github.com/magento-commerce/venia-sample-data-modules/pull/13

In the frontend and on the Contact Us Page accessible via the footer, check the following:
- Test and update the `Contact Us - Banner` CMS Block
- Test and update the `Contact Us - Sidebar` CMS Block
- Test without the CMS Blocks
- Test and close Backend PR: https://github.com/magento-commerce/venia-sample-data-modules/pull/13

## Checklist

-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: packages/venia-ui/lib/components/ContactPage/contactPage.js [hunk 1]
```diff
@@ -87,140 +93,169 @@ const ContactPage = props => {
...
-        </div>
+            </article>
+        </Fragment>
     );
 };
 
 ContactPage.propTypes = {
     classes: shape({
-        root: string,
+        loadingContainer: string,
```
Comment (JustinConabreeAbsolunet) [inline packages/venia-ui/lib/components/ContactPage/contactPage.js:249]: It seems as though this should stay as `root`? Not seeing `loadingContainer` changed elsewhere

**Review by JustinConabreeAbsolunet — CHANGES_REQUESTED** [review]
Looks good! One small potential issue

File: packages/venia-ui/lib/components/ContactPage/contactPage.js (hunk 1 above)
Comment (mikhaelbois) [reply packages/venia-ui/lib/components/ContactPage/contactPage.js:249]: I refreshed the classes list by their order of appearance in the file.
`root` is just below that. I can move it back at the top.

**Review by JustinConabreeAbsolunet — APPROVED** [review]
✅ LGTM!

**Pull request merged**